commands/ci/debug.js

```diff
@@ -0,0 +1,99 @@
+const cli = require('heroku-cli-util')
+const Dyno = require('heroku-run').Dyno
+const co = require('co')
+const api = require('../../lib/heroku-api')
+const git = require('../../lib/git')
+const source = require('../../lib/source')
+const TestRun = require('../../lib/test-run')
+
+// Default command. Run setup, source profile.d scripts and open a bash session
+const COMMAND = 'sprettur setup && for f in .profile.d/*; do source $f; done && bash'
+
+function* run (context, heroku) {
+  const coupling = yield api.pipelineCoupling(heroku, context.app)
+  const pipeline = coupling.pipeline
+  const commit = yield git.readCommit('HEAD')
+  const sourceBlobUrl = yield cli.action('Preparing source', co(function* () {
+    return yield source.createSourceBlob(commit.ref, context, heroku)
+  }))
+
+  const pipelineRepository = yield api.pipelineRepository(heroku, pipeline.id)
+  const organization = pipelineRepository.organization &&
+                       pipelineRepository.organization.name
+
+  // Create test run and wait for it to transition to `debugging`
+  const testRun = yield cli.action('Creating test run', co(function* () {
+    const run = yield api.createTestRun(heroku, {
+      commit_branch: commit.branch,
+      commit_message: commit.message,
+      commit_sha: commit.ref,
+      pipeline: pipeline.id,
+      organization,
+      source_blob_url: sourceBlobUrl,
+      debug: true
+    })
+
+    return yield TestRun.waitForStates(['debugging', 'errored'], run, { heroku })
+  }))
+
+  if (testRun.status === 'errored') {
+    cli.exit(1, `Test run creation failed whilst ${testRun.error_state} with message "${testRun.message}"`)
+  }
+
+  const appSetup = yield api.appSetup(heroku, testRun.app_setup.id)
+
+  const dyno = new Dyno({
+    heroku,
+    app: appSetup.app.id,
+    command: context.flags['no-setup'] ? 'bash' : COMMAND,
+    'exit-code': true,
+    'no-tty': context.flags['no-tty'],
+    attach: true,
+    env: 'HEROKU_SUPPRESS_LOGGING=true',
+    size: context.flags.size,
+    showStatus: false
+  })
+
+  try {
+    cli.log('Running setup and attaching to test dyno...')
+    dyno.start()
```

Comment:
Could we log a helpful message after this resolves? Say something like "Run `sprettur setup` to setup your tests or `sprettur run` to run them.".